client/components/create-account.jsx

```diff
@@ -37,13 +38,18 @@ export default class CreateAccount extends React.Component {
           username: ''
         });
         this.fileInputRef.current.value = null;
+        window.localStorage.setItem('username', this.state.username);
       })
       .catch(err => console.error(err));
     this.setState({ imgSrc: null });
     event.target.reset();
   }
 
   render() {
+    const currentUsername = window.localStorage.getItem('username');
```

Comment:
It's not good to grab things from localStorage in render. Instead send the user info up to App's state and have App pass it to ViewAccount. Also, app should swap from CreateAccount to ViewAccount instead of ViewAccount going inside of CreateAccount.